fe/fe-core/src/main/java/com/starrocks/common/InternalErrorCode.java

```diff
@@ -38,7 +38,9 @@ public enum InternalErrorCode {
     TOO_MANY_FAILURE_ROWS_ERR(102),
     CREATE_TASKS_ERR(103),
     TASKS_ABORT_ERR(104),
-    SLOW_RUNNING_ERR(105);
+    SLOW_RUNNING_ERR(105),
+    ICEBERG_CATALOG_ERROR(106),
```

Comment:
If use StarRocksConnectorException, there is no need to use error code, the error message is enough?

Reply:
modified it.